docs/fastdds/library_overview/library_overview.rst

```diff
@@ -3,45 +3,3 @@
 Library Overview
 ================
 
-
```

Comment:
Do not modify this file in this PR, it's taken care of in a different one

Reply:
I know, problem was I started working on it. I don't know what was its status before my modifications so I deleted everything. What should I do to go back to the start?

Reply:
On the diff of you commit here in GitHub you can see what was here before you started. Just put that